src/openassetio-python/package/openassetio/hostApi/Manager.py

```diff
@@ -430,6 +436,23 @@ def getWithRelationship(
 
         @param context Context The calling context.
 
+        @param successCallback Callback that will be called for each
+        successful relationship query. It will be given the
+        corresponding index of the entity reference in
+        `entityReferences` along with a list of entity references for
+        entities that have the specified relationship. If there are no
+        relations, an empty list will be supplied. The callback will be
+        called on the same thread that initiated the call to
+        `getWithRelationship`.
+
+        @param errorCallback Callback that will be called for each
+        failed relationship query. It will be given the
+        corresponding index of the entity reference in `entityReferences`
```

Comment:
[Consider][Minor]
Suggesting that we do this to document parameters. It will have no real changes, as all `\p` is is an alias for `\c`, which just displays typewriter font. However this does document that this intends to refer to a parameter rather then just an abstract variable.
```suggestion
        corresponding index of the entity reference in \p entityReferences
```

Reply:
Like it, but will `@p` work instead?  Bit more consistent

Reply:
Ill update these more generally (as we use `name` in other methods) once we have sign-off on the copy.

Reply:
Yeah, it's minor, (it would be better if it linked,) using `@p` is additional documentation language that makes it clearer that the following token is meant to refer to a parameter of the method in question, which I think is worth doing.

Reply:
Yeah I agree, assuming the read knows what @p means, then it's really useful to know that a term is a parameter.

Reply:
Changed all param references in `Manager` and `ManagerInterface` docs. We may have others in other files. https://github.com/OpenAssetIO/OpenAssetIO/pull/935/commits/2934bbaa991bf1c5f6a7c929799af7443b81c121